### PR TITLE
Updating tests to new master

### PR DESCRIPTION
**Pull request by mimmi20**
This PR will change tests to use the new master of browscap/browscap-php.

Actually builds on hhvm will fail because the job is killed. I asked Travis about that. They said the memory limit is reached. On a local hhvm instance the tests were finished successful.

One test is marked as skipped because it fails on PHP 7 when creating the zip file.

This PR was requested before with number #734.

### CODE REVIEW

**Comment by mimmi20** [comment]
need to be fixed

**Comment by asgrim** [comment]
@mimmi20 please don't merge anything that changes `browscap/browscap` to use `browscap/browscap-php` 3.x yet - I need to update the website as well before we can proceed

**Comment by mimmi20** [comment]
@asgrim Please merge this branch if you are ready for that.

This PR needs the optimize branch of browscap/browscap-php to be merged to get working successfull.

File: tests/BrowscapTest/UserAgentsTest.php [hunk 1]
```diff
@@ -264,10 +263,18 @@ public function testUserAgentsStandard($userAgent, $expectedProperties, $lite =
      * @param bool   $standard
      *
      * @throws \Exception
+<<<<<<< HEAD
```
Comment (asgrim) [inline tests/BrowscapTest/UserAgentsTest.php]: there's some merge conflict resolution tags here left over @mimmi20, please remove these

Comment (mimmi20) [inline tests/BrowscapTest/UserAgentsTest.php]: they are removed

**Comment by asgrim** [comment]
@mimmi20 thanks

**Pull request merged**